tools/SetupFlow/DevHome.SetupFlow/Views/InstallationNotesView.xaml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.SetupFlow.Views.InstallationNotesView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid Name="FullPage">
+        <!-- Extended TitleBar -->
+        <Grid x:Name="AppTitleBar"
+              Canvas.ZIndex="0"
+              Height="30"
+              IsHitTestVisible="True"
+              VerticalAlignment="Top">
+            <Image 
+                Source="{x:Bind DevHomeIconPath, Mode=OneWay}"
+                HorizontalAlignment="Left"
+                Width="16"
+                Height="20" 
+                Margin="10 5 5 0"/>
+            <TextBlock x:Name="AppTitleBarText"
+                VerticalAlignment="Center"
+                TextWrapping="NoWrap"
+                Style="{StaticResource CaptionTextBlockStyle}"
+                Text="{x:Bind PackageTitle, Mode=OneWay}"
+                Margin="35,5,5,0"/>
```

Comment:
Instead of setting a margin here and on the image, I think the more xaml-y thing is to set the margin on the AppTitleBar Grid, and then also set a "Spacing" on the grid so the Image and TextBlock are separated (I'd do 8 px since everything is supposed to be multiples of 4).

Reply:
I agree! I think there are several enhancements opportunities here. Because this change is coming in hot, I am using the same code in Dev Drive window. Ideally, I want the app title to be reusable so we don't have to reimplement it on each window (after this change we will have 3 copies: ShellPage.xaml, DevDriveView.xaml and InstallationNotesView.xaml) ☹️. That's not efficient and hard to maintain. To address that, I created an issue to eliminate redundancy and allow reusability. #1536